youtube_dl/extractor/tastytrade.py

```diff
@@ -33,7 +49,36 @@ def _real_extract(self, url):
             r'data-media-id=(["\'])(?P<code>(?:(?!\1).)+)\1',
             webpage, 'ooyala code', group='code')
 
-        info = self._search_json_ld(webpage, display_id, fatal=False)
+        info = {'id': None, 'title': None, 'description': None}
+
+        json_ld_info = self._search_json_ld(
+            webpage, display_id, default=None, fatal=False)
+
+        if (json_ld_info):
+            info = json_ld_info
+        else:
+            escaped_json_string = self._search_regex(
+                r'var episodeData = \$.parseJSON\("(?P<episode_json>.*)"\)',
+                webpage,
+                'episode json',
+                fatal=False,
+                group='episode_json'
+            )
+
+            if (escaped_json_string):
+                if sys.version_info[0] >= 3:
```

Comment:
The processing done here was to convert an escaped json string into an unescaped string so that it could be parsed properly. I'm open to suggestions for other ways of achieving the same goal.

Reply:
>I'm open to suggestions for other ways of achieving the same goal.

Did you read my message at all? `escaped_json_string.encode('utf-8').decode('unicode_escape')`.

Reply:
Ah, okay, I misunderstood. I thought you were saying that that had already been applied somewhere, or that it had been effectively applied already. Now I see that you're saying that it can done to the return from the method.